main/php_globals.h

```diff
@@ -80,6 +80,7 @@ struct _php_core_globals {
 	char *user_dir;
 	char *include_path;
 	char *open_basedir;
+	bool open_basedir_modified;
```

Comment:
I know @nielsdos did some struct packing at one time, but there was also one case where it wasn't super worth it to do it compared to keeping the relevant bits nearby. Might have been the globals struct

Reply:
Yeah. I prefer keeping related globals together, as it's easier to understand but also potentially improves cache locality. I'm not going to die on a hill for this though.

Reply:
I indeed did a bit of struct packing here too. When I did it I made sure that fields that were in the same cacheline were still in the same line after reordering. However, packing is not important here, this is fine as-is imo.